### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by heyshakya**
Update README.md with a 1-bit adder design with XLS HLS, OpenLane, and SKY130 PDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,11 @@ Run a simple inverter design thru the [OpenLane](https://github.com/The-OpenROAD
...
 
 ![mosfet layout](img/mosfet.png)
+
+### XLS adder with OpenLane [![colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/main/xls-adder-openlane.ipynb)
+
+[xls-adder-openlane.ipynb](xls-adder-openlane.ipynb)
+
+Run a simple 1-bit adder design using the [XLS](https://google.github.io/xls/) high level synthesis toolkit thru the [OpenLane](https://github.com/The-OpenROAD-Project/OpenLane/) GDS to RTL flow targeting the [open source SKY130 PDK](https://github.com/google/skywater-pdk/).
+
+![adder layout](https://drive.google.com/uc?export=view&id=1DYKbQg2kJJxHAZ5afZgVZU3-ZxO6lQnA)
```
Comment (proppy) [inline README.md:29]: can you add the image to the pull request instead?

**Comment by proppy** [comment]
@heyshakya thanks for the pull request! just a tiny comment.

File: README.md (hunk 1 above)
Comment (heyshakya) [reply README.md:29]: adder_layout

Comment (proppy) [reply README.md:29]: sorry, what I meant is adding it as part of the `img/` subdirectory along side your pull request.

**Comment by proppy** [comment]
Also pushed #17's proper fix.

**Pull request merged**